Dalamud/Game/ClientState/Actors/ActorTable.cs

```diff
@@ -56,37 +53,34 @@ public int Length
             }
         }
 
-        /// <inheritdoc/>
-        int IReadOnlyCollection<Actor>.Count => this.Length;
-
-        /// <inheritdoc/>
-        int ICollection.Count => this.Length;
-
-        /// <inheritdoc/>
-        bool ICollection.IsSynchronized => false;
-
-        /// <inheritdoc/>
-        object ICollection.SyncRoot => this;
-
-        private ClientStateAddressResolver Address { get; }
-
         /// <summary>
-        ///     Get an actor at the specified spawn index.
+        /// Get an actor at the specified spawn index.
         /// </summary>
         /// <param name="index">Spawn index.</param>
-        /// <returns><see cref="Actor" /> at the specified spawn index.</returns>
+        /// <returns>An <see cref="Actor"/> at the specified spawn index.</returns>
         [CanBeNull]
         public Actor this[int index]
         {
             get
             {
-                var ptr = this.GetActorAddress(index);
-                if (ptr != IntPtr.Zero)
-                {
-                    return this.CreateActorReference(ptr);
-                }
+                var address = this.GetActorAddress(index);
+                return this[address];
+            }
+        }
 
-                return null;
+        /// <summary>
+        /// Get an actor at the specified address.
+        /// </summary>
+        /// <param name="address">The actor address.</param>
+        /// <returns>An <see cref="Actor"/> at the specified address.</returns>
+        public Actor this[IntPtr address]
```

Comment:
Are you sure this is not confusing? Technically, you're not indexing anything.

Reply:
It could be. would you prefer a GetActor  or CreateActor(IntPtr)

Reply:
Well, we could just expose CreateActorReference too

Reply:
I just wanted to ask what the problem with making that public was, I think that'd be more sensible

Reply:
pushed